### DISCUSSION TITLE
Plus character (+) missing in SMS message text body

### DISCUSSION THREAD

**Discussion by bzivkovicveleri** [ibnux/Android-SMS-Gateway#39] [General]
Hello, I have a little problem - SMS sends perfectly, but it's missing a plus (+) character from body text:

**PHP code:**
$msgBody = "Message sent from +38599000001"; 
$sendTo = "+38599000002"; // this works
echo file_get_contents("https://sms.ibnux.net/?to=".urlencode($sendTo )."&text=".urlencode($msgBody)."&secret=$secret&time=$time&deviceID=".urlencode($deviceID));
**recieved SMS:** // missing plus sign
"Message sent from 38599000001"

**Comment by ibnux** [comment]
I will check it later

As I don't strip any messages so I can send unicode
